src/server/server_family.cc

```diff
@@ -147,6 +150,78 @@ class LinuxWriteWrapper : public io::WriteFile {
 
 }  // namespace
 
+bool IsValidSaveScheduleNibble(string_view time, unsigned int max, unsigned int min_len = 2) {
+  size_t digit_mask = std::pow(10, time.size() - 1);
```

Comment:
I had written the loop to work from the most significant digit to least significant mainly for cases like "1x" input for the hour as any value for x would be valid. Is there a better way to ensure the value is <= the maximum without converting to an integer since there could be wildcards?